index.js

```diff
@@ -0,0 +1,37 @@
+// import modules
+import Book from './modules/Book.js';
+import Storage from './modules/Storage.js';
+import UI from './modules/UI.js';
```

Comment:
PR linter checks are still not green and failing. So kindly fix the errors before asking for another review.

App navigation is confusing for a user. User when clicks on `Contact` link the form is open to add book. Kindly see the screenshots of navigation bar to understand the issue and resolve it.
![contact](https://user-images.githubusercontent.com/35504149/177211340-87c7047e-3e69-4473-b1d3-d8af2f8cf374.png)
![add](https://user-images.githubusercontent.com/35504149/177211358-27f67697-5441-426b-b5cf-ca3e4e8d094f.png)


![lists](https://user-images.githubusercontent.com/35504149/177211367-430a43d0-8dd0-4818-86e1-5b132d3460c4.png)